CHANGELOG.md

```diff
@@ -31,6 +31,8 @@
 
 #### Improvements
 
+- Unsupported types in `DataFrameReader.dbapi`(PuPr) are ingested as `StringType` now.
```

Comment:
what about base_driver and UDTF based ingestion?

Reply:
udtf ingestion share same process of acquire schema, so this change also applies to udtf ingestion.
base_driver does not have to_snow_type, when user want to use base_driver to ingest(when driver and dbms are not directly supported), they also need to provide custom_schema, so we don't need to apply this change to base_driver